docs/reactivity/integrate-backend-api.md

```diff
@@ -0,0 +1,77 @@
+# Building a reactive frontend with Oryx
+
+## Qualify the request
+
+Components, services and adapters are customizable. To ensure that a change in the requested data by a component can be used in lower layers, a best practice is to use a so-called qualifier in the application flow. The qualifier is a single object that will be passed fro application layer to application layer, and can be extended. The example below shows the use of the `ProductQualifier` in the ProductService.
```

Comment:
From one application layer to another? Or should we mention specific layers?

Reply:
We do have another docs on layers, and we've introduced layers quite a bit as part of this doc. When the layers docs are ready we can make a link here. Either way, I've reworded the text a bit, hope that helps.